training/checkpoint.py

```diff
@@ -1,6 +1,16 @@
 import os
 import torch
-
+from typing import Optional
+from unidecode import unidecode
+
+NVIDIA_ALPHABET = ['_', '-', '!', "'", '(', ')', ',', '.', ':', ';', '?', ' ',
```

Comment:
We already have a default alphabet defined at https://github.com/BenAAndrew/Voice-Cloning-App/blob/main/training/train.py#L35. Also, will this cause problems for non-English datasets or datasets that do not contain cmudict?

Reply:
When transferring embedding layers with different symbols, you need to know the original symbols and the new symbols since the index of the symbol is it's location in the layer.
NVIDIA_ALPHABET contains the alphabet in the exact order used by nvidia's pretrained model on their repo.

Reply:
OK Cool, that makes sense. Are we confident this won't cause problems for non-English datasets or datasets that do not contain cmudict?

Reply:
It will not cause problems. You can't initialize the embedding layer worse than scratch.

https://github.com/pytorch/pytorch/blob/1901c675e1cf9626a25d96bcc64b4952c64d56a2/torch/nn/modules/sparse.py#L148-L150
If you do not use this code then it will use literally random noise. This code cannot make the model worse than filling the embedding with random noise haha.

Reply:
https://github.com/CookiePPP/cookietts/tree/experimental/CookieTTS

Also to be clear, I've used similar code for my modules for over a year now. It works.

Reply:
Awesome! Just checking that it won't impact other use cases (like other languages) 😄